website/docs/r/secrets_sync_azure_destination.html.md

```diff
@@ -0,0 +1,84 @@
+---
+layout: "vault"
+page_title: "Vault: vault_secrets_sync_azure_destination resource"
+sidebar_current: "docs-vault-resource-secrets-sync-azure-destination"
+description: |-
+  Creates a Azure destination to synchronize secrets in Vault
+---
+
+# vault\_secrets\_sync\_azure\_destination
+
+Creates a Azure Key Vault destination to synchronize secrets in Vault. Requires Vault 1.16+.
+*Available only for Vault Enterprise*.
+
+~> **Important** All data provided in the resource configuration will be
+written in cleartext to state and plan files generated by Terraform, and
+will appear in the console output when Terraform runs. Protect these
+artifacts accordingly. See
+[the main provider documentation](../index.html)
+for more details.
+
+For more information on syncing secrets with Azure Key Vault, please refer to the Vault
+[documentation](https://developer.hashicorp.com/vault/docs/sync/azurekv).
+
+## Example Usage
+
+```hcl
+resource "vault_secrets_sync_azure_destination" "az" {
+  name                 = "az-dest"
+  key_vault_uri        = var.key_vault_uri
+  client_id            = var.client_id
+  client_secret        = var.client_secret
+  tenant_id            = var.tenant_id
+  custom_tags = {
+    "foo" = "bar"
+  }
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `namespace` - (Optional) The namespace to provision the resource in.
+  The value should not contain leading or trailing forward slashes.
+  The `namespace` is always relative to the provider's configured [namespace](/docs/providers/vault#namespace).
+
+* `name` - (Required) Unique name of the Azure destination.
+
+* `key_vault_uri` - (Optional) URI of an existing Azure Key Vault instance.
+  Can be omitted and directly provided to Vault using the `KEY_VAULT_URI` environment
+  variable.
+
+* `tenant_id` - (Optional) JSON-encoded credentials to use to connect to Azure.
+  Can be omitted and directly provided to Vault using the `AZURE_TENANT_ID` environment
+  variable.
```

Comment:
I think this is just an account-specific uuid, not JSON-encoded data.

Reply:
Fixed